api/src/main/java/org/openmrs/ConditionClinicalStatus.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+/**
+ * ConditionClinicalStatus enum class
```

Comment:
Class description should not be a mere repetition of the class name. Take a look this: https://github.com/openmrs/openmrs-core/commit/7411f5cce4eba1dbc83a89de699f240d124c5502